app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,18 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    if user = User.from_omniauth(request.env["omniauth.auth"])
+      session[:user_id] = user.id
+      redirect_to root_path
+    else
+      redirect_to root_path
+    end
+  end
+
+  def destroy
+    session.clear
```

Comment:
if we store something else in the sessions we might want to make this a session[:user_id] but this works right now

Reply:
We want to clear the entire session for the User when they sign out? I'm down if that is the route we go but the other option would be `session[user_id] = nil` so that we are still tracking behavior from that user.